images/alpine/jdk11/Dockerfile.alpine.jdk11-mvn3.6-mono5-nuget5.7

```diff
@@ -0,0 +1,24 @@
+ARG SNAPSHOT=""
+FROM strongboxci/alpine:jdk11-mvn3.6-mono5$SNAPSHOT
+
+# setup-nuget.sh
+ENV NUGET_VERSION=5.7.0
+ENV NUGET_PATH=/usr/local/bin/nuget-${NUGET_VERSION}.exe
+
```

Comment:
@steve-todorov I've renamed env variables here which will be used in strongbox/strongbox#1905.

Reply:
This is a **separate** task and needs to be done as part of a separate PR. :)

Reply:
Oops... Okay will revert change and rename env vars.